src/main/mal/Networking.mal

```diff
@@ -237,32 +230,32 @@ category Networking {
 
       // All the hidden attack steps below are hidden because they are just used for the internal mechanics of the ConnectionRules
       | attemptReverseReach @hidden
-        developer info: "Intermediate attack step."
+        developer info: "Intermediate attack step to allow for defenses and trigger bypasses."
         ->  reverseReach,
             bypassRestricted,
             bypassPayloadInspection
 
       & reverseReach @hidden
-        developer info: "Reverse reach is used to determine whether or not the attacker can be reached by the user."
-        ->  clientApplications().attemptReverseReach,
+        developer info: "Reverse reach is used to determine whether or not the attacker can be reached by the user. Reverse reach propagates via outgoing or bidirectional communications."
+        ->  senderApplications().attemptReverseReach,
```

Comment:
Looking on the old code, we had `clientApplications().attemptReverseReach` here and we now have `senderApplications().attemptReverseReach`. But isn't `receiverApplications` the equivalent of `clientApplications`? If that is the case then I messed up and renamed them the wrong way around :O

Reply:
Actually, I am ultra confused right now. Previously we had `let clientApplications = (applications \/ outApplications)` so that means that the `Applications` that are having an outbound `ConnectionRule` were the clients?
This became now: `let senderApplications = (applications \/ outApplications)`

Reply:
No, I think your changes are correct, reverse reach propagates towards sender applications, it goes the opposite direction as regular network connectivity.